src/utils/daos_hdlr.c

```diff
@@ -436,9 +436,9 @@ pool_query_hdlr(struct cmd_args_s *ap)
 		fprintf(stderr, "pool query failed: %d\n", rc);
 		D_GOTO(out_disconnect, rc);
 	}
-	D_PRINT("Pool "DF_UUIDF", ntarget=%u, disabled=%u\n",
+	D_PRINT("Pool "DF_UUIDF", ntarget=%u, disabled=%u, version=%u\n",
```

Comment:
In the sample output in DAOS-4771 it shows leader= included, between disabled= and version=. Does it need to be added here?

Reply:
When I included it in earlier iterations, I got a linter warning the line being too long, so I punted on including it for this tool because it's not clear that it's useful. I can remove it from the sample output.